app/lib/theme/CustomDiffFields.tsx

```diff
@@ -308,7 +444,8 @@ const CUSTOM_DIFF_FIELDS: Record<
           formContext?.oldData?.[props.name],
           uiSchema?.isMoney,
           uiSchema?.isPercentage,
-          uiSchema?.numberOfDecimalPlaces
+          uiSchema?.numberOfDecimalPlaces,
+          isAmendmentsAndOtherRevisionsSpecific
```

Comment:
This entire file has basically for each diff field a:
```
if (isAmendmentsAndOtherRevisionsSpecific) then ... else ...
```
I'm wondering if we should instead maintain separately an `AmendmentSpecificCustomDiffFields`, and load it where appropriate instead. It would improve readability and make the classes smaller. We could define an interface for it too.

I don't think we should make the change in this PR, but this could be captured in a Tech Debt card

Reply:
Totally agreed, the only thing that prevented me from doing this was the amount of duplication we will have between these two files.

Reply:
@pbastia captured in ticket: https://app.zenhub.com/workspaces/climate-action-secretariat-60ca4121764d710011481ca2/issues/gh/bcgov/cas-cif/1391